kotest-framework/kotest-framework-engine/src/jvmMain/kotlin/io/kotest/engine/config/applyConfigFromAutoScan.kt

```diff
@@ -22,7 +22,12 @@ internal actual fun applyConfigFromAutoScan(configuration: ProjectConfiguration)
       return
    }
 
-   println("Warning: Kotest autoscan is enabled. This means Kotest will scan the classpath for extensions that are annotated with @AutoScan. To avoid this startup cost, set autoscan to false by setting the system property 'kotest.framework.classpath.scanning.config.disable=true'. In 6.0 this value will default to true. For further details see https://kotest.io/docs/next/framework/project-config.html#runtime-detection")
+   println("Warning: Kotest autoscan is enabled. This means Kotest will scan the classpath for extensions that are "
+      + "annotated with @AutoScan. To avoid this startup cost, set autoscan to false by setting the system property "
```

Comment:
Grab the opportunity and clear up this wording as well?
```suggestion
      + "annotated with @AutoScan. To avoid this startup cost, disable autoscan by setting the system property "
```